### ISSUE TITLE
Top keys not working in Triad synths

### ISSUE THREAD

**Issue by mschack** [mschack/VCV-Rack-Plugins#23]
Just noticed this now.

**Comment by blackearth2014** [comment]
Yes. Also when you mistakenly connect 6x32 trigger sequencer to a 3 OSC 1/OCT port, VCVRack Crashes.

**Comment by mschack** [comment]
fixed, I hope...

**Comment by blackearth2014** [comment]
I will test

**Comment by blackearth2014** [comment]
The mixers look way better now with the improved graphics.
This is a screencapture of the old modules before you made the changes.

![thinktank_window1](https://user-images.githubusercontent.com/7956901/33226908-9dd4e848-d165-11e7-8c45-54f8f68e4077.png)

![thinktank_window2](https://user-images.githubusercontent.com/7956901/33226910-a725e35c-d165-11e7-8c88-fb406d23bfd1.png)
